src/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md

```diff
@@ -892,6 +892,10 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
 
 *  `php bin/magento cron:run` no longer processes items from the change log table multiple times. Previously, when you had more than 100000 new versions in the change log table, actions could be called several times for the same `entity id`.
 
+*  `php bin/magento setup:cron:run` command has now been removed. References to this in crontabs should be removed when upgrading.
+
+*  The `update/cron.php` file has been removed in Magento 2.4.0, if this file exists on your installation, it can be safely removed. References to this file in crontabs should also be removed when upgrading.
```

Comment:
```suggestion
*  The `update/cron.php` file has been removed in Magento 2.4.0, if this file exists on your installation, it can be safely removed. Also, remove references to this file in crontabs when upgrading.
```